src/senaite/core/upgrade/v02_05_000.py

```diff
@@ -129,3 +133,22 @@ def setup_catalogs(tool):
     setup_core_catalogs(portal)
 
     logger.info("Setup Catalogs [DONE]")
+
+
+def update_report_catalog(self):
+    """Update indexes in report catalog and add new metadata columns
+    """
+    logger.info("Update report catalog ...")
+    portal = api.get_portal()
+
+    # ensure new indexes are created
+    setup_catalog_mappings(portal)
+    setup_core_catalogs(portal)
+
+    # remove columns
+    del_column(REPORT_CATALOG, "getClientTitlegetClientURL")
```

Comment:
Oops, should be:

```python
del_column(REPORT_CATALOG, "getClientTitle")
del_column(REPORT_CATALOG, "getClientURL")
```

Reply:
No, it is actually correct. Note the missing `,` in the catalog.

Reply:
Oops, you are right